docs/dev-concepts/participant-authentication.md

```diff
@@ -0,0 +1,76 @@
+
+Anyone can verify the user's signature and consider it as an attestation that the `IdentityPublicKey` is the user's true proxy keys since only the user's blockchain account keys could have created the signature.
+
+## Prekeys
+
+A client also generates public and private prekeys for use with a user's `IdentityPublicKey`, or XMTP identity.
+
+A client prompts the user to sign their public prekey with their `IdentityPrivateKey`.
+
+<!--Below, looking for an accurate positioning for prekeys - as signing on behalf of an XMTP identity? Something like - the prekeys belong to, or as associated with, the XMTP identity? Is the idea that the prekeys - once signed by the identity keys - are the keys that are doing all the work? And that's why being able to rotate them in the future is important? Whereas, the identity keys sign the prekeys, but then aren't actively being used to encrypt and sign anything. And the blockchain account keys after signing the identity keys are REALLY out of the picture. I am seeing something like layers of defense - or obfuscation - or abstraction - is this how it is meant?-->
+
+The prekeys are used to encrypt invitations and sign encoded messages on behalf of the XMTP identity.
+<!--I don't explicitly mention the prekey use in the invitation encryption flows. I mention the public and private key bundles - when we talk about these bundles - are we specifically talking about the public and private prekeys? https://xmtp.org/docs/dev-concepts/invitation-and-message-encryption#invitation-encryption.-->
+
+<!--In the message encryption flows I show the sender signing the message with their prekey (I should specify private prekey). For the message decryption flow, I show the recipient verifying that message sender using the sender's public key from the message header - specifically, is this the public prekey? https://xmtp.org/docs/dev-concepts/invitation-and-message-encryption#message-encryption-->
+
+An XMTP identity may have multiple private prekeys. A client must be able to access every private prekey used by an XMTP identity to decrypt historical messages.
+
+There is always only one public prekey for an XMTP identity, which is the most current prekey.
+
+## Key generation and retrieval flow
+
+The following diagrams show how a client generates and retrieves identity keys and prekeys. The diagrams also show details of how a client encrypts and stores keys.
+
+Encrypted keys generated for a blockchain account are accessible by only the client apps authorized by the user (controller of the blockchain account). By design, XMTP and the XMTP network cannot access these encrypted keys. <!--I think we should remove this "By design..." sentence. The message in the previous sentence is clearer without it. By mentioning two things that cannot access the keys only opens up the question of what else can't access the keys. Also, what do we mean by XMTP? I believe we mean XMTP Labs.-->
```

Comment:
Agreed, the second sentence doesn't add much.

Reply:
Removed.